cellfinder_napari/detect.py

```diff
@@ -129,183 +80,130 @@ def widget(
+        filter_width: float,
+        threshold: int,
+        cell_spread: float,
+        max_cluster: int,
         classification_options,
-        Trained_model: Path,
+        trained_model: Path,
         misc_options,
-        Start_plane: int,
-        End_plane: int,
-        Number_of_free_cpus: int,
-        Analyse_local: bool,
-        Debug: bool,
+        start_plane: int,
+        end_plane: int,
+        number_of_free_cpus: int,
+        analyse_local: bool,
+        debug: bool,
         reset_button,
     ) -> List[napari.types.LayerDataTuple]:
         """
 
         Parameters
         ----------
-        Signal_image : napari.layers.Image
+        signal_image : napari.layers.Image
              Image layer containing the labelled cells
-        Background_image : napari.layers.Image
+        background_image : napari.layers.Image
              Image layer without labelled cells
         voxel_size_z : float
             Size of your voxels in the axial dimension
         voxel_size_y : float
             Size of your voxels in the y direction (top to bottom)
         voxel_size_x : float
             Size of your voxels in the x direction (left to right)
-        Soma_diameter : float
+        soma_diameter : float
             The expected in-plane soma diameter (microns)
         ball_xy_size : float
             Elliptical morphological in-plane filter size (microns)
         ball_z_size : float
             Elliptical morphological axial filter size (microns)
-        Ball_overlap : float
+        ball_overlap : float
             Fraction of the morphological filter needed to be filled
             to retain a voxel
-        Filter_width : float
+        filter_width : float
             Laplacian of Gaussian filter width (as a fraction of soma diameter)
-        Threshold : int
+        threshold : int
             Cell intensity threshold (as a multiple of noise above the mean)
-        Cell_spread : float
+        cell_spread : float
             Cell spread factor (for splitting up cell clusters)
-        Max_cluster : int
+        max_cluster : int
             Largest putative cell cluster (in cubic um) where splitting
             should be attempted
-        Trained_model : Path
+        trained_model : Path
             Trained model file path (home directory (default) -> pretrained weights)
-        Start_plane : int
+        start_plane : int
             First plane to process (to process a subset of the data)
-        End_plane : int
+        end_plane : int
             Last plane to process (to process a subset of the data)
-        Number_of_free_cpus : int
+        number_of_free_cpus : int
             How many CPU cores to leave free
-        Analyse_local : bool
+        analyse_local : bool
             Only analyse planes around the current position
-        Debug : bool
+        debug : bool
             Increase logging
         reset_button :
             Reset parameters to default
```

Comment:
I'm taking your word for it that the number of inputs in the docstring matches the number in the function

Reply:
Yea... opening the plugin in the GUI doesn't work if not.